### DISCUSSION TITLE
How to apply custom 4x4 shade to a brush when 'painting?'

### DISCUSSION THREAD

**Discussion by prahou** [Interlisp/medley#1105] [Q&A] [answered]
When in the 'paint' mode inside any window, holding mouse 3 and selecting SetShade provides a menu with 3 shades and a fourth option 4x4 shade, which opens a window for designing custom shades. 

But how does one apply the custom shade as the current brush shade?

Thank you.

**Comment by masinter** [comment, marked as answer]
Read the code...
It's all there

**Reply by prahou** [reply]
You have to close the custom shade window first. How about that!
